src/patternfly/components/Radio/examples/radio-checked-example.hbs

```diff
@@ -0,0 +1,4 @@
+{{#> radio}}
+  {{#> radio-input radio-input--attribute='id="radio3" name="exampleRadio4" checked'}}{{/radio-input}}
+  {{#> radio-label radio-label--attribute='for="radio3"'}}Radio 3 checked{{/radio-label}}
```

Comment:
I don't think the number in the label text is necessary anymore. We can just use "Radio," "Radio reversed," "Radio label wraps input," etc